test/unit/uk/gov/hmrc/testuser/services/TestUserServicesPEC.scala

```diff
@@ -40,20 +40,23 @@ class TestUserServiceSpec extends UnitSpec with MockitoSugar {
     val underTest = new TestUserService {
       override val generator: Generator = mock[Generator]
       override val testUserRepository: TestUserRepository = mock[TestUserRepository]
+      override val encryptionService: EncryptionService = mock[EncryptionService]
     }
     when(underTest.testUserRepository.createUser(Matchers.any[TestUser]())).thenAnswer(sameUser)
   }
 
   "createTestIndividual" should {
 
-    "Generate an individual and save it in the database" in new Setup {
+    "Generate an individual and save it with encrypted password in the database" in new Setup {
 
+      val encryptedPassword  = "encryptedPassword"
       given(underTest.generator.generateTestIndividual()).willReturn(testIndividual)
+      given(underTest.encryptionService.encrypt(testIndividual.password)).willReturn(encryptedPassword)
```

Comment:
Apparently the file name for this class is wrong.
`TestUserServiceSpec` instead of `TestUserServicesPEC`